File: cpp v2/Assignment.cpp
// File: Assignment.cpp
#pragma once
#include <string>

using namespace std;

class Assignment {
private:
    string title;
    string dueDate;

public:
    Assignment() {}
    Assignment(string t, string d) : title(t), dueDate(d) {}

    ~Assignment() {}

    string getTitle() const { return title; }
    void setTitle(string t) { title = t; }

    string getDueDate() const { return dueDate; }
    void setDueDate(string d) { dueDate = d; }
};

File: python/Code/Assignment.py
class Assignment:
    # Konstruktor untuk membuat objek Assignment baru
    def __init__(self, title, dueDate):
        self.__title = title
        self.__dueDate = dueDate

    # Mengambil judul tugas
    def getTitle(self):
        return self.__title

    # Mengambil tanggal deadline
    def getDueDate(self):
        return self.__dueDate

    # Mengubah judul tugas
    def setTitle(self, title):
        self.__title = title

    # Mengubah tanggal deadline
    def setDueDate(self, dueDate):
        self.__dueDate = dueDate
